terraform/azure/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "environment" {
+    type = string
+    description = "Name of the system or environment"
+    default = "dev"
+}
+
+variable "location" {
+    type = string
+    description = "Azure location of terraform server environment"
+    default = "Germany West Central"
+}
+
+variable "club" {
+    type = string
+    description = "The name of the club the installation is for"
+    default = "orso"
+}
+
+variable "backendconfig" {
+    description = "appsettings"
+    type = object({
```

Comment:
The variable definition looks good except that you know better what values are needed. I can check only the syntax.. may be you think of defining "defaults" to the entire backendconfig?

Reply:
done